src/games/strategy/triplea/delegate/MustFightBattle.java

```diff
@@ -2125,14 +2124,6 @@ private boolean isPartialAmphibiousRetreat() {
     return games.strategy.triplea.Properties.getPartialAmphibiousRetreat(m_data);
   }
 
-  private boolean isParatroopers(final PlayerID player) {
```

Comment:
There is a number of different things happening in this PR:
- rename from can not to cannot
- reformat
- methods moved
- static method call updates (e.g: `isParatroopers` -> `TechAttachment.isParatroopers`)

In the future pls try to keep PRs focused to one logical task if you can - make it really easy to review. So this could have potentially been 4 smaller / more focused PRs.